buildpipeline/dotnet-standard-win.json

```diff
@@ -1,11 +1,13 @@
 {
   "build": [
     {
+      "environment": {},
       "enabled": true,
       "continueOnError": false,
       "alwaysRun": false,
       "displayName": "Install Signing Plugin",
       "timeoutInMinutes": 0,
+      "refName": "Task1",
```

Comment:
I think you can remove these "refName" entries

Reply:
You could port https://github.com/dotnet/standard/commit/ada7bdbf69acf191abd7e1c31c253b49bb4abf4b#diff-4caa8c56eee8a6602df1fde15da88400 as well.